.github/workflows/cd.yml

```diff
@@ -81,6 +82,9 @@ jobs:
               });
             });
             return res.data.id
+        env:
+          REF_NAME: ${{ github.ref_name }}
+          REF: ${{ github.ref }}
```

Comment:
I think this approach is fine...  

 `actions/github-script` does make many of the github variables accessible through the context object -- so  env section would not be needed here, but I'm not sure if ref_name is included there so this seems like a decent and understandable way to do it.